src/notfication/notify.py

```diff
@@ -0,0 +1,28 @@
+from email.message import EmailMessage
+import smtplib
+import os
+import ssl
+# from dotenv import load_dotenv
+# load_dotenv()
+
+class Notify:
+    # __serviceAccEmail = os.getenv('SERVICE_EMAIL')
+    # __serviceAccPassword = os.getenv('SERVICE_PASSWORD')
```

Comment:
I see that you're referencing these variables later in the `email` method so why are they commented?